src/recensio/plone/content/review_article_collection.py

```diff
@@ -50,17 +50,21 @@ class ReviewArticleCollection(Item):
     """Content-type class for IReviewArticleCollection."""
 
     def formatted_authors(self):
+        # TODO This is here as a hint for the one implementing citations.
+        # Please remove this method and use IAuthors instead.
         authors_str = IAuthors(self).get_formatted_authors()
         return authors_str
 
     def getDecoratedTitle(self):
         args = {
-            "(Hg.)": translate(
+            "(Hg.)": api.portal.translate(
                 Message("label_abbrev_editor", "recensio", default="(Hg.)")
```

Comment:
We could also write:
```suggestion
                _("label_abbrev_editor", default="(Hg.)")
```